content/en/docs/reference/node/kubelet-files.md

```diff
@@ -114,6 +114,7 @@ The types of plugins that can place socket files into that directory are:
 
 - CSI plugins
 - DRA plugins
+
```

Comment:
> The kubelet looks for socket files created by device or storage plugins

Should be:
The kubelet looks for socket files created by DRA or storage plugins

Reply:
@bart0sh if we make that change, we'll need to teach what DRA and CSI are (in this page, I mean). The wording here is something I proposed to avoid needing to do that.

Reply:
@sftim I was mainly worrying about `device or storage plugins`, which may make users think that `/var/lib/kubelet/plugins_registry` is used by [device plugins](https://kubernetes.io/docs/concepts/extend-kubernetes/compute-storage-net/device-plugins/).

Reply:
changed wording